src/Compilers/VisualBasic/Test/Semantic/Semantics/Parenthesized.vb

```diff
@@ -138,10 +138,13 @@ BC30451: 'C2' is not declared. It may be inaccessible due to its protection leve
                                  ~~
 BC37259: Tuple must contain at least two elements.
     System.Console.WriteLine(GetType((System.String)))
-                                     ~~~~~~~~~~~~~~~
+                                                   ~
+BC30456: 'C2' is not a member of 'Type'.
```

Comment:
`GetType((C1).C2` is parsed as a member access with the expression being `GetType((C1)` and the member being `C2`. There is a missing token for the close-paren on `GetType` and `(C1)` is a 1-tuple type.

`GetType` returns a `System.Type`. That's the `Type`.

Previously, binding `(C1)` would produce a `ErrorTypeSymbol.UnknownResultType` so I think the binding would not continue. But now, we produce a tuple `(C1, ?)` which allows binding to continue.